src/VisualStudio/Core/Def/Implementation/Utilities/BrowserHelper.cs

```diff
@@ -59,7 +59,8 @@ private static bool TryStartBrowser(IServiceProvider serviceProvider, Uri uri)
 
         private static bool TryStartBrowser(IVsWebBrowsingService service, Uri uri)
         {
-            return ErrorHandler.Succeeded(service.CreateExternalWebBrowser((uint)__VSCREATEWEBBROWSER.VSCWB_ForceNew, VSPREVIEWRESOLUTION.PR_Default, uri.AbsoluteUri));
+            IVsWindowFrame unused;
+            return ErrorHandler.Succeeded(service.Navigate(uri.AbsoluteUri, 0, out unused));
         }
 
         private static void StartBrowser(Uri uri)
```

Comment:
Looks like we fallback to external browser if internal fails. Should we stop doing this now?